src/Database/Adapter/Postgres.php

```
@@ -1937,7 +1957,7 @@ protected function getFulltextValue(string $value): string
     protected function getSQLType(string $type, int $size, bool $signed = true, bool $array = false, bool $required = false): string
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$signed'. (undefined)

(UnusedFormalParameter)